Formula/spotbugs.rb

```diff
@@ -35,6 +35,7 @@ def install
     EOS
     system "javac", "HelloWorld.java"
     system "jar", "cvfe", "HelloWorld.jar", "HelloWorld", "HelloWorld.class"
-    assert_match /M V EI.*\nM C UwF.*\n/, shell_output("#{bin}/spotbugs -textui HelloWorld.jar")
+    output = shell_output("#{bin}/spotbugs -textui HelloWorld.jar")
+    assert_match /M V EI.*\nM C UwF.*\n/, output
```

Comment:
you can avoid the \n's by using /m at the end.

```
assert_match /M V EI.*M C UwF/m, output
```
should work

Reply:
Since each bug shows up on a separate line, I specifically wanted to catch the fact the 2 "bugs" were found (if that's ok)

Reply:
sure that's fine.